Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM quay.io/operator-framework/ansible-operator:v1.3.0
+
+COPY requirements.yml ${HOME}/requirements.yml
+USER 0
+RUN update-crypto-policies --set LEGACY && pip3 install pyopenssl
+RUN ansible-galaxy collection install -r ${HOME}/requirements.yml \
+ && chmod -R ug+rwx ${HOME}/.ansible
+RUN yum install git -y
+RUN git clone --single-branch --branch master https://github.com/noironetworks/acc-provision.git
```

Comment:
Could we make the branch an argument (with default being master)? That way we don't have to keep updating this file for every release. The build job will pass the branch name as a build arg.